Fw/Logger/test/ut/LoggerRules.cpp

```diff
@@ -60,51 +65,51 @@ void LogGood::action(MockLogging::FakeLogger& truth) {
             correct = "No args";
             break;
         case 1:
-            Fw::Logger::log("One arg: %lu", ra[0]);
-            correct.format("One arg: %lu", ra[0]);
+            Fw::Logger::log("One arg: %llu", ra[0]);
```

Comment:
Good catch! Here is a suggestion:

```suggestion
            Fw::Logger::log("One arg: %" PRI_U64 ", ra[0]);
```

The C format specifiers `%u`, `%lu`, and `%llu` mean `unsigned`, `unsigned long`, and `unsigned long long`. They should bind only to those exact C/C++ types and not to any other types such as `U64`, which in general have different representations on different platforms. For those other types, we should use the PRI macros, which are provided by the compiler and are correct for each platform. The original code also looks incorrect to me (it may be a holdover from before we had the PRI macros).